[2022-06-26 09:08:33,186] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T09:08:30.062430+00:00 [queued]>
[2022-06-26 09:08:33,211] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T09:08:30.062430+00:00 [queued]>
[2022-06-26 09:08:33,212] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 09:08:33,213] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-06-26 09:08:33,214] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 09:08:33,226] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): postgreSQL_getTable> on 2022-06-26T09:08:30.062430+00:00
[2022-06-26 09:08:33,237] {standard_task_runner.py:52} INFO - Started process 10626 to run task
[2022-06-26 09:08:33,243] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'SDG_Enrique_V0', 'postgreSQL_getTable', '2022-06-26T09:08:30.062430+00:00', '--job-id', '193', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/company_forecasting.py', '--cfg-path', '/tmp/tmpdm92531g', '--error-file', '/tmp/tmp5ooa8izo']
[2022-06-26 09:08:33,246] {standard_task_runner.py:77} INFO - Job 193: Subtask postgreSQL_getTable
[2022-06-26 09:08:33,304] {logging_mixin.py:104} INFO - Running <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T09:08:30.062430+00:00 [running]> on host 5e8dc4b25004
[2022-06-26 09:08:33,357] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=enrique-liebana@outlook.com
AIRFLOW_CTX_DAG_OWNER=Enrique
AIRFLOW_CTX_DAG_ID=SDG_Enrique_V0
AIRFLOW_CTX_TASK_ID=postgreSQL_getTable
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T09:08:30.062430+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-26T09:08:30.062430+00:00
[2022-06-26 09:08:33,371] {base.py:74} INFO - Using connection to: id: postgres_conection. Host: host.docker.internal, Port: 5432, Schema: data_company, Login: airflow, Password: XXXXXXXX, extra: None
[2022-06-26 09:08:49,014] {company_forecasting.py:77} INFO - Backup Company data in : /raw_data/company.csv
[2022-06-26 09:08:49,015] {python.py:118} INFO - Done. Returned value was: None
[2022-06-26 09:08:49,034] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=SDG_Enrique_V0, task_id=postgreSQL_getTable, execution_date=20220626T090830, start_date=20220626T090833, end_date=20220626T090849
[2022-06-26 09:08:49,075] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-26 09:08:49,134] {local_task_job.py:146} INFO - Task exited with return code 0
